README.md

```diff
@@ -37,7 +37,7 @@ $ rails g settings:install
 Or use a custom name:
 
 ```bash
-$ rails g settings:install SiteConfig
+$ rails g settings:install Setting
```

Comment:
Please do not change here, `SiteConfig` is a special name

Reply:
Oh, got you. I thought I could use any name here and it will generate a setting for specific purposes. Ok, let me change it really quick...